src/index.ts

```diff
@@ -2,22 +2,27 @@ import { watch } from 'chokidar';
 import pLimit from 'p-limit';
 import { cliLog, CliOptions, getCliOptions, printCliHelpMessage } from './cli';
 import { buildApplication } from './buildApplication';
+import { nextOnPagesVersion } from './utils';
 
 const limit = pLimit(1);
 
 const cliOptions = getCliOptions();
 runNextOnPages(cliOptions);
 
 function runNextOnPages(options: CliOptions): void {
-	cliLog('@cloudflare/next-on-pages CLI');
+	if (options.version) {
+		// eslint-disable-next-line no-console -- for the version lets simply print it plainly
+		console.log(nextOnPagesVersion);
+		return;
+	}
+
+	cliLog(`@cloudflare/next-on-pages CLI v.${nextOnPagesVersion}`);
```

Comment:
I think it's much nicer to just show the version with `--version` so I moved this log below it (also this log shows the version already so it would be a bit awkward to include it)